Models/User.js

```diff
@@ -46,6 +46,14 @@ const UserSchema = new Schema({
   standups: {
     type: Array,
   },
+  configs: {
+    medium_mode: {
```

Comment:
maybe rename `medium_mode` to `notification_destination` ?

- `email`
- `slack`
- `web` ...

Reply:
noted on this!